doc/api/esm.md

```diff
@@ -95,7 +95,7 @@ until the root of the volume is reached.
 ```
 
 ```bash
-# In same folder as above package.json
+# In same folder as the preceding package.json.
```

Comment:
```suggestion
# In the same folder as the preceding package.json.
```

Are we writing full sentences in comments? Otherwise `In same folder as preceding package.json` feels more appropriate.

Reply:
I don't think we have a standard for bash script comments, so I'll go with your preferred article-free phrasing.